src/Catalyst.Core.Modules.Ledger.Tests/IntegrationTests/LedgerKvmTests.cs

```diff
@@ -200,130 +232,174 @@ public void Should_Deploy_Code_On_Code_Entry()
+            delta.PublicEntries[1].GasLimit = 100_000L; // has to be enough for intrinsic gas 21000 + call
+            delta.PublicEntries[2].GasLimit =
+                3_000_000L;                             // has to be enough for intrinsic gas 21000 + CREATE + code deposit
+            delta.PublicEntries[3].GasLimit = 100_000L; // has to be enough for intrinsic gas 21000 + call
+            delta.PublicEntries[4].GasLimit = 200_000;
+            delta.PublicEntries[4].TargetContract = contractAddress2.Bytes;
+
+            delta.PublicEntries[0].Signature = delta.PublicEntries[0]
+               .GenerateSignature(_cryptoContext, _senderPrivateKey, _signingContext);
+            delta.PublicEntries[1].Signature = delta.PublicEntries[1]
+               .GenerateSignature(_cryptoContext, _senderPrivateKey, _signingContext);
+            delta.PublicEntries[2].Signature = delta.PublicEntries[2]
+               .GenerateSignature(_cryptoContext, _senderPrivateKey, _signingContext);
+            delta.PublicEntries[3].Signature = delta.PublicEntries[3]
+               .GenerateSignature(_cryptoContext, _senderPrivateKey, _signingContext);
+            delta.PublicEntries[4].Signature = delta.PublicEntries[4]
+               .GenerateSignature(_cryptoContext, _senderPrivateKey, _signingContext);
+
             RunDeltas(delta);
 
             var migrationAccount = _stateProvider.GetAccount(contractAddress1);
             migrationAccount.Balance.Should().Be(0);
-            _stateProvider.GetCode(migrationAccount.CodeHash).Should().Equal(Bytes.FromHexString("608060405260043610610062576000357c0100000000000000000000000000000000000000000000000000000000900463ffffffff1680630900f01014610067578063445df0ac146100aa5780638da5cb5b146100d5578063fdacd5761461012c575b600080fd5b34801561007357600080fd5b506100a8600480360381019080803573ffffffffffffffffffffffffffffffffffffffff169060200190929190505050610159565b005b3480156100b657600080fd5b506100bf610241565b6040518082815260200191505060405180910390f35b3480156100e157600080fd5b506100ea610247565b604051808273ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff16815260200191505060405180910390f35b34801561013857600080fd5b506101576004803603810190808035906020019092919050505061026c565b005b60008060009054906101000a900473ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff163373ffffffffffffffffffffffffffffffffffffffff16141561023d578190508073ffffffffffffffffffffffffffffffffffffffff1663fdacd5766001546040518263ffffffff167c010000000000000000000000000000000000000000000000000000000002815260040180828152602001915050600060405180830381600087803b15801561022457600080fd5b505af1158015610238573d6000803e3d6000fd5b505050505b5050565b60015481565b6000809054906101000a900473ffffffffffffffffffffffffffffffffffffffff1681565b6000809054906101000a900473ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffff
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Split this 1541 characters long line (which is greater than 200 authorized).](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4600656)